service/controller/v15/resource/deployment/master_deployment.go

```diff
@@ -250,7 +249,7 @@ func newMasterDeployments(customObject v1alpha1.KVMConfig) ([]*extensionsv1.Depl
 								Lifecycle: &apiv1.Lifecycle{
 									PreStop: &apiv1.Handler{
 										Exec: &apiv1.ExecAction{
-											Command: []string{"/qemu-shutdown"},
+											Command: []string{"/qemu-shutdown", fmt.Sprintf("%s/v1/defer/", key.ShutdownDeferrerListenAddress(customObject))},
```

Comment:
I think it would be good to have a key function for the whole path. Looking below I was wondering why we have a path here and below not. Should we hard code the path in the shutdown script here as well and only configure the address? Then we would had some symmetry.

Reply:
I created key function for now. I think think we need to improve the whole deferrer architecture a little bit later (to get rid of shell scripts and having importable Go code as Pawel suggested in another PR).

Reply:
Alright. Then let's refactor it later.